src/python/bot/untrusted_runner/host.py

```diff
@@ -150,6 +153,12 @@ def wrapped(*args, **kwargs):
       try:
         return func(*args, **kwargs)
       except grpc.RpcError as e:
+        # For timeouts, which aren't fatal errors, resurface the right
```

Comment:
Should add testcase_manager.TargetNotFoundError as well i think ?

Reply:
Want to keep this more generic as with `TimeoutError`. `testcase_manager.TargetNotFoundError` is pretty specific to a single RPC, so keeping it there.

Reply:
SGTM.